MarcusBarnes/mik
# Work on #354.

**Github issue**: (#354)

# What does this Pull Request do?

Adds a shutdown hook script to clear out the MIK temporary directory.

# What's new?

A shutdown hook script to clear out the MIK temporary directory.

# How should this be tested?

1. Check out the issue-354 branch.
1. In any toolchain, register the new script in your .ini file's [WRITER] section: `[WRITER] shutdownhooks[] = "php extras/scripts/shutdownhooks/delete_temp_directory.php"`
1. Run MIK. After it finishes running, look in the directory defined in your `[FETCHER] temp_directory` setting. It should be empty.

# Interested parties

@bondjimbond
